database/migrations/2021_06_30_062631_create_outcome_data_view.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOutcomeDataView extends Migration
```

Comment:
Will this work for new deployment where the table being used to create view will be empty when migrations are run ?